packages/@orchidui-vue/src/Builder/Form/OcFormBuilder.vue

```diff
@@ -47,8 +47,8 @@ const props = defineProps({
 
 const emit = defineEmits(["onUpdate"]);
 
-const onUpdate = (form, value) => {
-  emit("onUpdate", form, value);
+const onUpdate = (form, value, fieldName) => {
```

Comment:
we can access fieldName using form.name | we have string and object type for name.

Reply:
Thank you @kewcoder you're right/ updated 🙏